src/main/java/org/jitsi/videobridge/osgi/ConfigurationActivator.java

```diff
@@ -37,19 +37,11 @@
     @Override
     public void start(BundleContext bundleContext)
     {
-        ConfigurationService cfg = LibJitsi.getConfigurationService();
-        if (cfg != null)
-        {
-            bundleContext.registerService(
-                    ConfigurationService.class.getName(),
-                    cfg,
-                    null);
-            logger.info("Registered the LibJitsi ConfigurationService in OSGi.");
-        }
-        else
-        {
-            logger.warn("Failed to register the Configuration service.");
-        }
+        bundleContext.registerService(
```

Comment:
I wonder if this should now move to jicoco. Presumably jicofo will want to do the exact same thing.

Reply:
At this rate I don't know when we'll even get to porting Jicofo, but I agree and can move it there if/when do the Jicofo work.